src/cli/cli.ts

```diff
@@ -36,14 +36,14 @@ process.on('unhandledRejection', (reason: any, _promise: any) => {
 })
 
 const registriesByName: {[name: string]: (kit: ContractKit) => Registry} = {
-	"mento": (kit: ContractKit) =>	new RegistryMento(kit),
-	"ubeswap":	mainnetRegistryUbeswap,
-	"sushiswap":	mainnetRegistrySushiswap,
-	"mobius":	mainnetRegistryMobius,
-	"moola":	mainnetRegistryMoola,
-	"moola-v2":	mainnetRegistryMoolaV2,
-	"savingscelo":	mainnetRegistrySavingsCELO,
-	"celodex":	mainnetRegistryCeloDex,
+	"mento":       (kit: ContractKit) => new RegistryMento(kit),
+	"ubeswap":     mainnetRegistryUbeswap,
+	"sushiswap":   mainnetRegistrySushiswap,
+	"mobius":      mainnetRegistryMobius,
+	"moola":       mainnetRegistryMoola,
+	"moola-v2":    mainnetRegistryMoolaV2,
+	"savingscelo": mainnetRegistrySavingsCELO,
+	"celodex":     mainnetRegistryCeloDex,
```

Comment:
aligned these with spaces so that it always lines up with fixed-width and regardless of tab sizes